R/call.R

```diff
@@ -83,30 +97,30 @@ lang <- function(.fn, ..., .ns = NULL) {
   }
 
   if (!is_null(.ns)) {
-    .fn <- new_language(sym_namespace, pairlist(sym(.ns), .fn))
+    .fn <- new_call(sym_namespace, pairlist(sym(.ns), .fn))
   }
 
-  new_language(.fn, as.pairlist(dots_list(...)))
+  new_call(.fn, as.pairlist(dots_list(...)))
 }
-#' @rdname lang
+#' @rdname call2
 #' @param head A [callable][is_callable] object: a symbol, call, or
 #'   literal function.
 #' @param tail A [node list][pairlist] of arguments.
 #' @export
-new_language <- function(head, tail = NULL) {
+new_call <- function(head, tail = NULL) {
```

Comment:
This should probably be `@keyword internal` even if it means documenting it somewhere else. Very very few people will need to use this.